changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Next
+
+- Added `numpy>=2.0` support.
```

Comment:
Would suggest to say that this only applies for Python versions 3.10 and upwards

Reply:
addressed